src/main/java/com/webank/wecross/Generator.java

```diff
@@ -3,24 +3,28 @@
 import com.webank.wecross.config.StubManagerConfig;
 import com.webank.wecross.stub.StubFactory;
 import com.webank.wecross.stubmanager.StubManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.context.ApplicationContext;
 import org.springframework.context.annotation.AnnotationConfigApplicationContext;
 
 public class Generator {
-
+    private static final Logger logger  = LoggerFactory.getLogger(Generator.class);
```

Comment:
Adding log is a good idea, but no need to remove printlns that can Inform user clearly in terminal.

Reply:
It's worth noting that you should run "./gradlew goJF" to format java code before committing.